packages/integration-sdk-core/src/types/config.ts

```diff
@@ -86,6 +86,7 @@ export interface InvocationConfig<
    *
    */
   ingestionConfig?: IntegrationIngestionConfigFieldMap;
+  stepWrapper?: StepWrapperFunction<TStepExecutionContext>;
```

Comment:
I'd probably add a doc string for the new parameter and what it could be used for.